repos/system_upgrade/el7toel8/libraries/config/version.py

```diff
@@ -137,15 +137,15 @@ def current_version():
     :return: The tuple contains release name and version value.
     :rtype: (string, string)
     """
-    facts_messages = api.consume(OSReleaseFacts)
+    facts_messages = api.consume(IPUConfig)
```

Comment:
Everyactor has the configuration included. You are not supposed to consume it. `api.current_actor().configuration.os_release`. From this point, the check doesn't make sense probably so much as well. As well, the original lib is wrong probably as it cannot use consume without check that actor is consuming the msg. As well, Import is not needed in such case.

Reply:
ok